clients/tfchain-client-go/price_test.go

```diff
@@ -0,0 +1,25 @@
+package substrate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrice(t *testing.T) {
+	cl := startLocalConnection(t)
+	defer cl.Close()
+
+	price, err := cl.GetTFTPrice()
+	require.Greater(t, int(price), 0)
+	require.NoError(t, err)
```

Comment:
I assume that if there is an error the price will be 0, right? So then this test will fail on the line 14. It will then give a wrong message to the developer. He will start looking into the code and checking why it is returning 0 while the developer should be looking into the returned error. I think we should always check the error before checking the result of the function. Can you swap these two lines and do the same below?